zoxide/add.go

```diff
@@ -3,18 +3,19 @@ package zoxide
 import (
 	"fmt"
 	"os/exec"
-	"path"
+	"path/filepath"
 )
 
 func Add(result string) error {
-	if !path.IsAbs(result) {
-		return fmt.Errorf("can't add relative %q path to zoxide", result)
+	p, err := filepath.Abs(result)
```

Comment:
Perhaps we could use determine path instead of just asserting the result is an absolute path. For now we make the caller responsible for providing a valid result string.

Reply:
Not sure if I am clear on the ask here but it sounds similar to the description of what `path.Abs` does...

> // Abs returns an absolute representation of path.
// If the path is not absolute it will be joined with the current
// working directory to turn it into an absolute path. The absolute
// path name for a given file is not guaranteed to be unique.
// Abs calls Clean on the result.

Reply:
I see, I can do a cleanup PR later to consolidate the logic.